lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -57,6 +57,18 @@ def handle(self, *args, **options):
         if not course_id:
             raise CommandError("You must specify a course-id")
 
+        def update_user_whitelist(username, add=True):
```

Comment:
I think we want to eventually pull get_user_from_identifier up further, so while this is a good question, I'm not sure making it a class method makes sense now.